### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matthewkotila**

### CODE REVIEW

File: genai-perf/README.md [hunk 1]
```diff
@@ -99,6 +99,8 @@ you'll need to install the Perf Analyzer binary:
 
 ```bash
 pip install tritonclient
+
+sudo apt update && sudo apt install -y --no-install-recommends libb64-0d
```
Comment (ganeshku1) [inline genai-perf/README.md:103]: Is the sudo expected here?

Comment (matthewkotila) [reply genai-perf/README.md:103]: Yes, sudo is generally needed when running apt as non-root user.

Comment (ganeshku1) [reply genai-perf/README.md:103]: As far as i recall we did not had that in the past version of the doc.
Approving this change as i anticipating we will be mostly reverting this change after we release 24.11.

**Review by ganeshku1 — APPROVED** [review]

File: genai-perf/README.md (hunk 1 above)
Comment (matthewkotila) [reply genai-perf/README.md:103]: We might have left it off accidentally. See here from the official ubuntu documentation showing that sudo is needed for apt:

https://ubuntu.com/server/docs/package-management#installing-packages

**Pull request merged**